fsharp-backend/src/LibExecution/Interpreter.fs

```diff
@@ -33,10 +35,12 @@ let withGlobals (state : ExecutionState) (symtable : Symtable) : Symtable =
 let rec eval' (state : ExecutionState) (st : Symtable) (e : Expr) : DvalTask =
   // Design doc for execution results and previews:
   // https://www.notion.so/darklang/Live-Value-Branching-44ee705af61e416abed90917e34da48e
-  // TODO document is either gone or hidden behind login
+  // TODO remove link from code or avail document - it is either gone or hidden behind login
   let sourceID id = SourceID(state.tlid, id)
   let incomplete id = DIncomplete(SourceID(state.tlid, id))
 
+  /// WHATISTHIS?
+  /// "if we're previewing some execution, ...?"
```

Comment:
I'm having a hard time understanding the purpose of, and describing, this function. Could you help? @pbiggar 
Full fn:
https://github.com/darklang/dark/blob/662e99b8cb70a48b1019503d8dba772f323b0f07/fsharp-backend/src/LibExecution/Interpreter.fs#L44-L50

Reply:
Thanks!